index.html

```diff
@@ -307,7 +307,7 @@ <h5 class="card-title">Outreach</h5>
 <h5 class="card-title">How to use UPI </h5>
 </a>
 <ul>
-<li>an Awareness drive was organised by us and we educated people how to use UPI and what are the benifits of using it</li>
+<li>A Awareness drive was organised by us and we educated people how to use UPI and what are the benifits of using it</li>

```

Comment:
Make the first letter of every sentence in uppercase, and rest all in lowercase (adapt this for every card).